examples/source/user-consent/Server_Side_User_Consent_Flow.html

```diff
@@ -0,0 +1,112 @@
+<!---
+
+teaserImage: '/static/samples/img/teaser/geolocation-based_consent_flow.jpg'
+author: micajuine-ho
+
+--->
+
+<!--
+  ## Introduction
+
+  Sometimes is necessary to rely on external servers for consent due to needing to sync user consent between multiple platforms. This sample demonstrates
+  how you can use `amp-consent` to achieve this. In this sample we'll build a
```

Comment:
please link to the component docs the first time you mention amp-consent in a sample